loadtest/user/userentity/user.go

```diff
@@ -96,6 +100,76 @@ func (ue *UserEntity) Connect() <-chan error {
 	return ue.wsErrorChan
 }
 
+// DownloadFile will download a url into a buffer
+func downloadFile(url string) ([]byte, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	if resp.StatusCode != http.StatusOK {
+		return nil, errors.New("File not found: " + url)
+	}
+
+	defer resp.Body.Close()
+
+	return ioutil.ReadAll(resp.Body)
+}
+
+func extractLinks(data []byte) []string {
+	tokenizer := html.NewTokenizer(strings.NewReader(string(data)))
+	links := []string{}
+	for {
+		switch tokenizer.Next() {
+		case html.ErrorToken:
+			// End of the document, we're done
+			return links
+		case html.StartTagToken:
+			tag, _ := tokenizer.TagName()
+			nodeType := ""
+			nodeValue := ""
+			if string(tag) != "script" {
```

Comment:
Any reason we are downloading only scripts and not css/images/other stuff ? The browser will download all that's there.